priv/datasets/wine_data.csv

```diff
@@ -0,0 +1,179 @@
+class,alcohol,malic_acid,ash,alcalinity_of_ash,magnesium,total_phenols,flavanoids,nonflavanoid_phenols,proanthocyanins,color_intensity,hue,od280/od315_of_diluted_wines,proline
```

Comment:
I'm a bit apprehensive of a `/` in a column name. Might be the R user in me, but it feels destined to break _something_ (even though it does appear to work fine). Perhaps `od280_or_od315_of_diluted_wines`?

Reply:
After a little research I found out it is actually a ratio to measure protein concentration. So I ended up with `od280_over_od315_of_diluted_wines`. Let me know if you agree.